knapsack/Python/knapsack.py

```diff
@@ -0,0 +1,35 @@
+def knapsack(v, w, n, W):
+    """
+    Function solves 0/1 Knapsack problem
+    Function to calculate frequency of items to be added to knapsack to maximise value
+    such that total weight is less than or equal to W
+    :param v: set of values of n objects, first element is useless because relevant values are assumed to begin from index 1
+    :param w: set of weights of n objects, first element is useless because relevant weights are assumed to begin from index 1
+    :param n: number of items
+    :pram W: maximum weight allowd by knapsack
+    """
+    m = [[0 for j in range(W+1)] for i in range(n+1)]
+    for j in range(W+1):
+        m[0][j] = 0
+
+    for i in range(1, n+1):
+        for j in range(W+1):
+            if w[i] > j:
+                m[i][j] = m[i-1][j]
+            else:
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpzpm74cqf/knapsack/Python/knapsack.py
+++ b/tmp/tmpzpm74cqf/knapsack/Python/knapsack.py
@@ -27,7 +27,7 @@
     w = [0, 5, 6, 2, 3, 4]
     n = 5
     W = 8
-    print("Maximum value possible in given weight limit: ", 
+    print("Maximum value possible in given weight limit: ",
         str(knapsack(v, w, n, W)))
 
 
```